tripy/pyproject.toml

```diff
@@ -46,7 +46,6 @@ test = [
   "pytest-virtualenv==1.7.0",
   "pytest-cov==4.1.0",
   "jax[cuda12_local]==0.4.23",
-  "jaxlib==0.4.23+cuda12.cudnn89",
```

Comment:
Does `jax` always pull in the right `jaxlib`? I don't remember exactly why we added this but I think it was because we sometimes getting the wrong version.

Reply:
It does seem to work in our testing.